rosetta-source/src/main/rosetta/product-common-schedule-type.rosetta

```diff
@@ -175,18 +175,18 @@ type ObservationDate: <"Specifies a single date on which market observations tak
 
 type ObservationTerms: <"Class containing terms that are associated with observing a price/benchmark/index across either single or multiple observations. ">
 
-    pricingTime BusinessCenterTime (0..1) <"Defines time in respect to a business calendar location that the price/benchmark/index is observed">
-    pricingTimeType TimeTypeEnum (0..1) <"The enumerated values to specify points in the day when option exercise and valuation can occur.">
+    observationTime BusinessCenterTime (0..1) <"Defines time in respect to a business calendar location that the price/benchmark/index is observed">
+    observationTimeType TimeTypeEnum (0..1) <"The enumerated values to specify points in the day when option exercise and valuation can occur.">
     informationSource FxSpotRateSource (0..1) <"The information source where a published or displayed market rate will be obtained, e.g. Telerate Page 3750.">
```

Comment:
@Oblongs 
suggest adding : 

these 2 attributes in type **ObservationTerms** : 

- observationMethod **DeterminationMethodEnum (0..1)** <"The enumerated values to specify the method according to which an amount or a date is determined.">
- priceObservation **PriceSchedule (0..1)** <"Record the numerical value of the Price effectively observed for the Observable defined in observablePrice attribute."> 
    [metadata address]

and this condition :

**condition** ObservationIsDatedValue: <"DatedValue must be used for the purpose of defining the date of the observation together with related value.">
     **_if_** priceObservation exists 
     _**then**_ priceObservation ->datedValue exists